workers/pull_request_worker/pull_request_worker/worker.py

```diff
@@ -677,6 +677,10 @@ def query_pr_meta(self, head, base,  pr_id):
 
             self.pr_meta_id_inc = int(result.inserted_primary_key[0])
             self.results_counter += 1
+
+            if new_head[0]['repo']:
+                self.query_pr_repo(new_head[0]['repo'], 'head', self.pr_meta_id_inc)
```

Comment:
@parthsharma2 : ... or, ignoring my other comment, is this where we are adding meta information for the source repo and the base repo? What table is the fork going in if so?

Reply:
The `query_pr_meta()` method adds the HEAD & BASE metadata to the `pull_request_meta` table. The repositories associated with HEAD & BASE are then added to the `pull_request_repo` table by the `query_pr_repo()` method.